xbee/tornado/tests/test_base.py

```diff
@@ -7,6 +7,10 @@
 
 Tests the XBeeBase superclass module for XBee API conformance.
 """
+
+import pytest
+pytest.importorskip("tornado")
```

Comment:
I'd rather not force the pytest runner to make this work. We can use the builtin [unittest `skipUnless`](https://docs.python.org/3/library/unittest.html#unittest.skipUnless) to achieve what we want like so:

```python
from xbee.tornado import has_tornado

@unittest.skipUnless(has_tornado)
class TestReadFromDevice(AsyncTestCase):
```